src/SearchParameters/index.js

```diff
@@ -1,36 +1,48 @@
 'use strict';
 
-var keys = require('lodash/keys');
 var intersection = require('lodash/intersection');
 var forOwn = require('lodash/forOwn');
-var isNaN = require('lodash/isNaN');
-var isEmpty = require('lodash/isEmpty');
-var isEqual = require('lodash/isEqual');
-var isUndefined = require('lodash/isUndefined');
-var isFunction = require('lodash/isFunction');
 
 var defaults = require('lodash/defaults');
 var merge = require('lodash/merge');
 
 var find = require('../functions/find');
 var valToNumber = require('../functions/valToNumber');
+var objectHasKeys = require('../functions/objectHasKeys');
 var omit = require('../functions/omit');
 
 var filterState = require('./filterState');
 
 var RefinementList = require('./RefinementList');
 
 /**
- * like _.find but using _.isEqual to be able to use it
+ * isEqual, but only for arrays of base elements, or base elements
+ * @param {number|Array<number>} a first element
+ * @param {number|Array<number>} b second element
+ */
+function isEqualNumArr(a, b) {
```

Comment:
We call it `isEqual*Num*Array` but it actually works with all primitive types like the doc you added says it.
There might be a consistency in the naming there.
I don't have a better naming suggestion 🐵 maybe `isShallowEqualArray`

Reply:
We could make this function work only for arrays, call it something like `isArrayEqual` and leave `a === b` apart. Since this is just an identity, we can use the identity on top of `isArrayEqual` whenever the old `isEqual` was used.

Reply:
called it `isEqualNumericRefinement`